migrations/20230318135354_rooms.up.sql

```diff
@@ -1,8 +1,8 @@
 -- Add up migration script here
 CREATE TABLE rooms(
-  room_id varchar PRIMARY KEY,
-  name varchar NOT NULL UNIQUE,
-  category varchar NOT NULL,
-  slots integer not null,
-  foreign key (category) references categories(category_id)
+  d text PRIMARY KEY,
```

Comment:
Did you mean "id"? Typo I assume
```suggestion
  id text PRIMARY KEY,
```

Reply:
Oh, yes. Rooms was unused yet.